### ISSUE TITLE
Builder Pattern

### ISSUE THREAD

**Issue by hibob224** [Muddz/StyleableToast#1]
Would be good to use a builder pattern for creating a styled toast from code. So instead of the README example of 

```  
StyleableToast st = new StyleableToast(this, "Updating profile", Toast.LENGTH_SHORT);
st.setBackgroundColor(Color.parseColor("#ff5a5f"));
st.setTextColor(Color.WHITE);
st.setIcon(R.drawable.ic_autorenew_black_24dp);
st.spinIconAnimation();  
st.setAlpha(StyleableToast.MAX_VISIBILTY);
st.show();
```

You could do something like 
```
new StyleableToast(this, "Example", Toast.LENGTH_SHORT)
	.setBackgroundColor(Color.parseColor("#ff5a5f"))
	.setTextColor(Color.WHITE)
	.setIcon(R.drawable.ic_autorenew_black_24dp)
	.spinIconAnimation()
	.setAlpha(StyleableToast.MAX_VISIBILITY)
	.show();
```

**Comment by PDDStudio** [comment]
@hibob224 I've opened a PR for your request.

**Comment by Muddz** [comment]
@PDDStudio  Thanks. I will look at it, but I can't merge/accept it yet, because I'm applying for a developer job and they want to look at some things I have done on Github and Playstore. 

I also just made some small changes today with the code and you forked the library before I made these small changes, just to let you know

**Comment by PDDStudio** [comment]
@Muddz no problem, you can merge it whenever you want. I'll merge your changes on my fork's master branch and update my pr.

**Comment by Muddz** [comment]
Implemented in 1.0.6
